index.html

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
You have removed all code, please add back removed code so it works with dark mode include all the code